cnf-tests/testbinaries/oslat-runner/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"flag"
+	"os/exec"
+	"strings"
+	"time"
+
+	"k8s.io/klog"
```

Comment:
do we want to use klog (more deps) or the stdlib `log` package is sufficient here?

Reply:
IIUC when you want to write to the file and to the stderr under the `log` you will need to set the output via ` SetOutput` to use multiple writers, it not too hard to do, but why re-invent the wheel when all deps anyway already placed under the vendor

Reply:
it's fine if we already have the deps